Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "SBML"
 uuid = "e5567a89-2604-4b09-9718-f5f78e97c3bb"
 authors = ["Mirek Kratochvil <miroslav.kratochvil@uni.lu>", "LCSB R3 team <lcsb-r3@uni.lu>"]
-version = "0.5.4"
+version = "0.5.5"
```

Comment:
please keep the version bumps separate from other commits. Patch releases are okay but chances are there might be more discussion related just to the version numbering (people depend on this).

Reply:
The goal of including the version bump in the pull request is to allow registering the new release immediately after: [release early, release often](https://en.wikipedia.org/wiki/Release_early,_release_often) 🙂 it's also much easier to judge within a pull request whether changes are breaking or not, or whether they introduce new publicly facing features, instead of looking back to a bunch of unrelated changesets.